File: sample/fib.c
#include<stdlib.h>
#include<stdio.h>
#include<unistd.h>


unsigned long fib(int n)
{
    if(n == 0){
       return 0;
    }else if(n == 1){
        return 1;
    }else{
        return fib(n - 2) + fib(n - 1);
    }
}


int main(int argc, char* argv[])
{
    int n = atoi(argv[1]);
    printf("%lu\n", fib(n));
}

File: sample/fib.py
# -*- coding:utf-8 -*-
def fib(n):
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fib(n - 2) + fib(n - 1)


if __name__ == '__main__':
    import sys
    n = int(sys.argv[1])
    print(fib(n))
